docs/usage/self-hosted-configuration.md

```diff
@@ -63,6 +63,41 @@ But before you disable templating completely, try the `allowedPostUpgradeCommand
 
 ## allowScripts
 
+## allowedHeader
+
+This is option particularly useful when a registry employs a specific authentication system not already covered by Renovate's standard credential handling in `hostRules`.
+By default, all header starting with "X-" are allowed, but you can permit additional header using this option.
```

Comment:
```suggestion
By default, all headers starting with "X-" are allowed.
If needed, you can allow additional headers with the `allowedHeader` option.
```

Reply:
it seems headers was right, should we use headers again?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

Reply:
Should I change it everywhere? What about allowedHeaders, should I revert that as well?